templates/create-frontend-plugin/docs/features/rbac.md

```diff
@@ -0,0 +1,129 @@
+        />
+    );
+    };
+
+    export const PermissionsCard = ({ entityReference }: PermissionsCardProps) => {
+    const { data, loading, retry, error } =
+        usePermissionPolicies(entityReference);
+    const [permissions, setPermissions] = React.useState<PermissionsData[]>();
+    const permissionResult = usePermission({ // check if the user has required permissions
+        permission: policyEntityUpdatePermission,
+        resourceRef: policyEntityUpdatePermission.resourceType,
+    });
+
+    // ... additional code ... //
+
+    const actions = [
+        {
+            icon: getRefreshIcon,
+            tooltip: 'Refresh',
+            isFreeAction: true,
+            onClick: () => {
+                retry.permissionPoliciesRetry();
+                retry.policiesRetry();
+            },
+        },
+        {
+            icon: () => getEditIcon(permissionResult.allowed, entityReference), // reference hook
+            tooltip: !permissionResult.allowed ? 'Unauthorized to edit' : 'Edit',
+            isFreeAction: true,
+            onClick: () => {},
+        },
+    ];
+
+    // ... additional code ... //
+```
+
+
+#### `RequirePermission` to block elements from rendering
+
+```ts
+    export const ClusterInfoCard = () => {
+    const { data } = useCluster();
+
+    if (!data) {
+        return null;
+    }
+
+    // ... Additional code ... //
+    return (
+        <RequirePermission permission={ocmEntityReadPermission}> // Block access to table
+            <TableCardFromData data={data} title="Cluster Info" nameMap={nameMap} />
+        </RequirePermission>
+    );
+```
+
+
+
+
+## References
+[OCM front-end plugin](https://github.com/janus-idp/backstage-plugins/tree/main/plugins/ocm)
+[RBAC front-end plugin](https://github.com/janus-idp/backstage-plugins/tree/main/plugins/rbac)
+[Topology front-end plugin](https://github.com/janus-idp/backstage-plugins/tree/main/plugins/topology)
+
+[Youtube video on permissions framework](https://www.youtube.com/watch?v=BDoQhegtw6E)
+
+Concepts:
+https://backstage.io/docs/permissions/concepts
+
+Basic permissions:
+Examples:
+- https://github.com/backstage/backstage/blob/master/plugins/catalog-common/src/permissions.ts
+- https://github.com/backstage/backstage/blob/master/plugins/jenkins-common/src/permissions.ts
+- https://github.com/backstage/backstage/blob/master/plugins/devtools-common/src/permissions.ts
+- https://github.com/backstage/backstage/blob/master/plugins/kubernetes-common/src/permissions.ts
+
+Condition permissions:
+Examples:
+- https://github.com/backstage/backstage/blob/master/plugins/catalog-backend/src/permissions/conditionExports.ts
+- https://github.com/backstage/backstage/blob/c9ddf111a70fbbf286abe13081b67fc7719108d4/plugins/playlist-backend/src/permissions/conditions.ts#L20
+- https://github.com/backstage/backstage/blob/master/plugins/scaffolder-backend/src/service/rules.ts
+- Upstream Documentation
+https://backstage.io/docs/permissions/custom-rules
```

Comment:
Here as well:
```suggestion
Condition permissions:

Examples:

- https://github.com/backstage/backstage/blob/master/plugins/catalog-backend/src/permissions/conditionExports.ts
- https://github.com/backstage/backstage/blob/c9ddf111a70fbbf286abe13081b67fc7719108d4/plugins/playlist-backend/src/permissions/conditions.ts#L20
- https://github.com/backstage/backstage/blob/master/plugins/scaffolder-backend/src/service/rules.ts
- Upstream Documentation
https://backstage.io/docs/permissions/custom-rules
```

![image](https://github.com/user-attachments/assets/553abec0-5441-44a7-b43b-f858e61f0795)

Can you also format them as markdown links?
The last point maybe one link?